routes/html/index.js

```diff
@@ -1,20 +1,27 @@
 const Users = require("../../models/Users");
 const Games = require("../../models/Games");
 const sequelize = require("../../config/connection");
+const { withAuth } = require("../../utils/auth");
 
 const router = require("express").Router();
 
 router.get("/", (req, res) => {
   const loggedIn = !!req.user;
   if (loggedIn) {
     // render the dashboard
-    res.render("dashboard", { loggedIn });
+    const userDBname = req.user.firstName;
+    const userDBusername = req.user.username;
+    res.render("dashboard", { userDBname, userDBusername, loggedIn });
   } else {
     // render the landing page
     res.render("landingpage");
```

Comment:
Shouldn't this redirect to the landing page?
You have two routes redirecting to the same screen

Reply:
Dashboard is page where user can access their profile link to their page

Landing page is our logo/banner page